### ISSUE TITLE
Problem formatting .svelte files

### ISSUE THREAD

**Issue by jakob-fuchs2** [jonlabelle/SublimeJsPrettier#271]
I have the following setup.

Prettier config:

```json
{
	"useTabs": true,
	"singleQuote": true,
	"trailingComma": "none",
	"printWidth": 100,
	"bracketSpacing":false
}
```

SublimeJSPrettier config:
```json
{
	"auto_format_on_save": false,
	"auto_format_on_save_requires_prettier_config": true,
	"auto_format_on_save_excludes": ["*/node_modules/*"],
	"custom_file_extensions": ["svelte"],
	"disable_tab_width_auto_detection": true,
	"disable_prettier_cursor_offset": true,
}
```

Additionally I have the "svelte" and "lsp-svelte" packages installed.

This is how I want the code to be formatted (and how VSCode and PHPStorm format it with the same prettier config with their respective prettier plugins, same if I run prettier from the command line):

```svelte
{#if hasPagination}
	<TablePagination
		{pageIndex}
		{pageCount}
		totalCount={serverSide ? $serverItemCount : $rows.length}
		{hasPreviousPage}
		{hasNextPage}
	/>
{/if}
```

When I format the code it ends up producing invalid syntax + removes the indentation after svelte structures like `#if` and `#each`:

```svelte
{#if hasPagination}
<TablePagination
	{pageIndex}
	{pageCount}
	totalCount="{serverSide"
	?
	$serverItemCount
	:
	$rows.length}
	{hasPreviousPage}
	{hasNextPage}
/>
{/if}
```

I appreciate any hints, most likely just a simple mistake on my part.

**Comment by jonlabelle** [comment]
Try telling the plugin to only use prettier options found in *prettier config files*. In your [JsPrettier settings](https://github.com/jonlabelle/SublimeJsPrettier/blob/778cda45c212f28054cad3c4aa01002c5a646e4f/JsPrettier.sublime-settings#L265), add the following line:

```json
 "additional_cli_args": { "--config-precedence": "file-override" }

**Comment by jakob-fuchs2** [comment]
Thanks a lot. I got it working now.

For future readers:

First I had to set the `node_path` in the SublimeJSPrettier package settings, most likely because I'm using [Volta](https://volta.sh/) to manage node versions.

Next I had to add the following line to my `.prettierrc` file:

```json
"overrides": [{ "files": "*.svelte", "options": { "parser": "svelte" } }]
``` 
And finally I had to add the line mentioned above to the SublimeJSPrettier package settings:

```json
 "additional_cli_args": { "--config-precedence": "file-override" }
```
